ui/src/components/info-page/terms-of-service-page/terms-of-service-page.tsx

```diff
@@ -0,0 +1,64 @@
+import { LoadingSpinner } from 'design-system/components/loading-spinner/loading-spinner'
+import { ReactElement, useEffect, useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+import styles from '../info-page.module.scss'
+import markdown from './terms-of-service.md'
+
+export const TermsOfServicePage = () => {
+  const [markdownContent, setMarkdownContent] = useState<string>()
+
+  useEffect(() => {
+    const loadContent = async () => {
+      const response = await fetch(markdown)
+      const markdownContent = await response.text()
+      setMarkdownContent(markdownContent)
+    }
+
+    loadContent()
+  })
+
+  if (!markdownContent) {
+    return (
+      <div className={styles.wrapper}>
+        <div className={styles.loadingWrapper}>
+          <LoadingSpinner />
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.container}>
+        <ReactMarkdown
+          className={styles.content}
+          components={{
+            ol: (props) => {
+              const start = props.start ?? 1
+              const children = (props.children as ReactElement[]).filter(
+                (element) => element.type === 'li'
+              )
+
+              // TODO: Can we find a better way to detect what is a term title?
```

Comment:
This is perhaps not an optimal way to detect what is a term title (this assumes all lists with one child is a term title). I did it to avoid modifying the markdown export, it works for now.

Reply:
Once the Terms are finalized, we can customize the markdown file in the repo (or in the database). But I understand that this is the easiest way to keep the terms up-to-date until then.